packages/core/src/oidc/utils.ts

```diff
@@ -27,19 +27,21 @@ const isOrigin = (value: string) => {
   }
 };
 
+export const validateCorsAllowedOrigins = (value: unknown) => {
```

Comment:
should CORS all Uppercase?

Reply:
camelcase is acceptable.

<img width="744" alt="image" src="https://user-images.githubusercontent.com/10594507/163096265-fad00ea9-c6d2-4d11-ae65-a0186274f0bf.png">

See Notion [doc](https://www.notion.so/silverhand/Common-36449f7172514044967e5dbe537b87e9#3deb4db5eb7846b28ce2d61a376a0082)

Reply:
See coding [convention](https://www.notion.so/silverhand/Common-36449f7172514044967e5dbe537b87e9#3deb4db5eb7846b28ce2d61a376a0082)

<img width="744" alt="image" src="https://user-images.githubusercontent.com/10594507/163095488-6b97a4a5-53ae-4dad-a426-acbd9b91078a.png">